tests/integration/test_reader_integrations.py

```diff
@@ -5,6 +5,12 @@
 import sys
 import unittest
 
+try:
+    import MDAnalysis
+    MDANALYSIS = True
+except ImportError:
+    MDANALYSIS = False
+
 
```

Comment:
Why is this, and the other logic changes in this file, a part of this PR?

Reply:
MDAnalysis does not provide wheels, and it's not compatible with the newest clang versions, so compiling will fail outright. This causes problems for our build because we run tests on CI that depend on MDAnalysis (the integration tests). The newest clang releases happened after our last release of freud, and we only build with clang when we build OS X wheels at release, so we simply hadn't run into this problem yet. Since I identified the issues when modifying the wheel building process to work with scikit-build, I had to make these changes to test the whole thing. Ideally it wouldn't be here, but even if I did it in a separate PR it would immediately need to be merged here, so I just left it as part of this PR.